### DISCUSSION TITLE
EditorHTML - Variavel option

### DISCUSSION THREAD

**Discussion by syspone** [balta-io/forum#251] [Tira Dúvidas] [answered]
@andrebaltieri  nas aulas do editorHTML nao entendi a criação da variavel option? poderia me da um help.

``` var option = short.Parse(Console.ReadLine());```

**Comment by andrebaltieri** [comment, marked as answer]
Boa tarde, @cmsoouza, como vai?
Essa variável foi criada para armazenar a opção que o usuário quer realizar no sistema.

**Reply by syspone** [reply]
blz professor
